client/utils/index.js

```diff
@@ -8,3 +8,13 @@ export const getAddToCartVariationParams = ( key ) => {
 
 	return wcAddToCartVariationParams[ key ];
 };
+
+/**
+ * Check if APMs are deprecated. Meaning it is past Oct 31st, 2024 and the legacy checkout is enabled.
+ *
+ * @return {boolean} Whether APMs are deprecated.
+ */
+export const areAPMsDeprecated = () =>
+	new Date() > new Date( '2024-10-31' ) &&
+	// eslint-disable-next-line camelcase
+	wc_stripe_settings_params.is_upe_checkout_enabled !== '1';
```

Comment:
What do you think about having a single place for this logic and just passing a `are_apms_deprecated` param from the backend in `wc_stripe_settings_params`?

Reply:
Yeah, that's smarter! Done ✅